src/main/kotlin/racing/model/Car.kt

```diff
@@ -0,0 +1,14 @@
+package racing.model
+
+class Car {
+    fun run() = getMoving()
+
+    companion object {
+        private fun getMoving(): Int {
+            var moving: Int = (0..9).random()
+            moving = if (moving >= 4) moving
+            else 0
+            return moving
```

Comment:
굳이 moving 변수의 값을 재할당할 필요는 없을 것 같아요. 🤔 
그렇다면 moving 변수는 val로 선언할 수 있을 것 같아요. 😃
(불변 변수는 꼭 `val`로 변수를 선언해주세요. 😊)


```suggestion
            val moving = (0..9).random()
            return if (moving >= 4) moving
                else 0
```